### ISSUE TITLE
Missing pipeline's env and cmd info from help message

### ISSUE THREAD

**Issue by MangelMaxime** [slaveOftime/Fun.Build#46]
Running the help command for the following pipeline only shows `-stageCommand` and `STAGE_ENV`. It should also show the information about `-pipelineCommand` and `PIPELINE_ENV`.

```fs
pipeline "bug-demo" {
    description "Check cmd info build style"
    // You can use whenCmd CE for more complex situation.
    whenCmd {
        shortName "-pipelineCommand"
    }
    whenEnv {
        name "PIPELINE_ENV"
    }
    stage "condition demo" {
        noStdRedirectForStep
        failIfIgnored
        whenAll {
            // You can use whenCmd CE for more complex situation.
            whenCmd {
                shortName "-stageCommand"
            }
            whenEnv {
                name "STAGE_ENV"
            }
        }
        echo "here we are"
    }
    runIfOnlySpecified
}
```

![CleanShot 2023-09-04 at 11 12 58](https://github.com/slaveOftime/Fun.Build/assets/4760796/1c372629-ab24-46c1-a3be-b80a3706f00e)

**Comment by albertwoo** [comment]
This related to the new feature which will combine top level conditions with && rule.

Should be fixed in 0.4.6.

**Comment by MangelMaxime** [comment]
I confirm it, thank you :)
